code/style.css

```diff
@@ -1,4 +1,308 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: rgb(204, 204, 204);
+  color: rgb(0, 0, 0);
+  font-weight: normal;
+}
+
+body{
+  margin: 0px;
+  font-family: 'Raleway', sans-serif;
+}
+
+/*Header and navbar*/
+
+.flex-container {
+  height:75px;
+  background-color:darkgrey;
+  color:black;
+  font-size: 25px;
+  display:flex;
+  align-items: center;
+  justify-content: space-evenly;
+  flex:wrap;
+}
+
+.logo{
+  height:150px;
+}
+
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: inline;
+}
+
+a{
+  text-decoration: none;
+  font-size: 30px;
+  font-weight: bold;
+  padding:15px;
+  color:black;
+}
+
+a:hover {
```

Comment:
Regarding the requirements: you added some hover effect on meaningful parts, great!